cpu/cortexm_common/ldscripts/cortexm.ld

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2017 Inria
+ * Copyright (C) 2018 Kaspar Schleiser <kaspar@schleiser.de>
```

Comment:
Since the original author is @kYc0o, I would leave the 2017 Inria copyright on top and put your personal under. I saw that in other places.